src/Compilers/CSharp/Portable/Symbols/NamedTypeSymbol.cs

```diff
@@ -1473,6 +1473,8 @@ ISymbol INamedTypeSymbol.AssociatedSymbol
         /// </summary>
         INamedTypeSymbol INamedTypeSymbol.TupleUnderlyingType => this.TupleUnderlyingType;
 
+        bool INamedTypeSymbol.IsComImport => IsComImport;
```

Comment:
alternatively, i can just make the existing IsComImport public.  Let me know what you guys prefer.